rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1120,7 +1120,7 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	};
 
 	// Events
-	connect(lib_mode_bg, static_cast<void(QButtonGroup::*)(int)>(&QButtonGroup::buttonClicked), on_lib_button_clicked);
+	connect(lib_mode_bg, static_cast<void(QButtonGroup::*)(QAbstractButton *)>(&QButtonGroup::buttonClicked), on_lib_button_clicked);
```

Comment:
These are not necessary.
In 5.15 the cast will be entirely ommited and relaxed by a "proper" connect.